website/content/docs/connect/intentions.mdx

```diff
@@ -97,6 +97,9 @@ accepted.
 An intention source or destination may also be the special wildcard
 value `*`. This matches _any_ value and is used as a catch-all.
 
+Wildcards can be used for service name matching. In Consul <EnterpriseAlert inline />
```

Comment:
```suggestion
Wildcards can be used for service name matching. In Consul Enterprise
```
We've typically only used these labels on section headings, or right after variables names in the API/CLI config options, but AFAIK not in the middle of a sentence.

Reply:
Do we support just the any-character wildcard? Would there every need be a need to specify something like `**/*-service` or `service_?` ?

Reply:
```suggestion
You can use a wildcard to match service names. If you are using Consul Enterprise, you can also use a wildcard to match a namespace.
```